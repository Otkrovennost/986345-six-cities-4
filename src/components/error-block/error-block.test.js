import React from "react";
import renderer from "react-test-renderer";
import ErrorBlock from "./error-block.jsx";

it(`Should NoResults render correctly`, () => {
  const tree = renderer
    .create(<ErrorBlock />).toJSON();

  expect(tree).toMatchSnapshot();
});
