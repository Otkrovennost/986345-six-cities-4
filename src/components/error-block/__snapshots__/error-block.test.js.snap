// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NoResults render correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#2219B2",
      "boxShadow": "0 10px 20px #554DD8",
      "color": "white",
      "fontSize": "24px",
      "position": "fixed",
      "right": 0,
      "textAlign": "center",
      "top": 0,
      "width": "100%",
    }
  }
>
  Sorry! No comment send. Check your internet connection.
</div>
`;
